README.md

```diff
@@ -188,149 +476,514 @@ Options:
+              {
+                populationId: 'ipp-population-id',
+                populationType: 'initial-population',
+                criteriaExpression: 'Initial Population',
+                result: true
+              },
+              {
+                populationId: 'numer-population-id',
+                populationType: 'numerator',
+                criteriaExpression: 'Numerator',
+                result: true
+              },
+              {
+                populationId: 'denom-population-id',
+                populationType: 'denominator',
+                criteriaExpression: 'Denominator',
+                result: true
+              },
+              {
+                populationId: 'measure-obs-population-id',
+                criteriaReferenceId: 'denom-population-id',
+                populationType: 'measure-observation',
+                criteriaExpression: 'daysObservation',
+                result: true,
+                observations: [2]
+              }
+            ]
+          }
+        ]
+      }
+    ]
+  }
+];
+```
 
-To find your VSAC API key, sign into [the UTS homepage](https://uts.nlm.nih.gov/uts/), click on `My Profile` in the top right, and copy the `API KEY` value from the `UMLS Licensee Profile`.
+From these results, it can be seen that `"episode-1"` has a duration of `1` day, and `"episode-2"` has a duration of `2` days. The results are also indicating the `populationId` property on each `populationResult`, as well as the `criteriaReferenceId` property on each measure observation `populationResult`. In the case of measure observations, it is common to assign an `id` property to each population in the `Measure` resource, as well as use the [cqfm-criteriaReference extension](https://build.fhir.org/ig/HL7/cqf-measures/StructureDefinition-cqfm-criteriaReference.html) on the measure observation population in order to indicate which population that function is observing (denominator in this example). Doing so allows consumers of the above result to easily identify which populations the measure observation results are referring to, which is especially useful if there are multiple measure observations defined.
 
-### TypeScript
+## `meta.profile` Checking
 
-`fqm-execution` exports custom-defined TypeScript interfaces used within the code to allow for easy integration into other TypeScript projects. The TypeScript files defining these interfaces can be found [here](https://github.com/projecttacoma/fqm-execution/tree/master/src/types).
+`fqm-execution` can be configured to use a [trusted environment with cql-exec-fhir](https://build.fhir.org/ig/HL7/cqf-measures/StructureDefinition-cqfm-criteriaReference.html), where resources returned as the result of an ELM Retrieve expression will only be included if they have a `meta.profile` list that contains the canonical URL of the profile being asked for by the Retrieve. To enable this environment, use the `trustMetaProfile` calculation option during calculation:
```

Comment:
I think the link here might be incorrect (links to the criteriaReference extension that is referenced in the previous section)

Reply:
Nah this was intentional. It's an exercise to the reader to find the correct link